docs/guides/admin/docs/upgrade.md

```diff
@@ -22,7 +22,9 @@ As part of removing mentions of the old name Matterhorn, all database table name
 with `oc_`.  This requires an database schema update. As with all database migrations, we recommend to make a database
 backup before attempting the upgrade.
 
-You can find the database upgrade script at `docs/upgrade/4_to_5/mysql5.sql`.
+You can find the database upgrade script in `docs/upgrade/4_to_5/`. There are two scripts which are essentially doing
```

Comment:
If this is going into develop, shouldn't this be 5_to_6?  Or maybe we should reword this to fit a more generic format?  For example:

You can find the database upgrade scripts in `docs/upgrade/X_to_Y/`.  As an example, to upgrade from Opencast 4 to Opencast 5 you would use the scripts in `4_to_5`.

Reply:
Good catch. It's fixing a 5.x bug and should go into `r/5.x`. I'll change the target branch…